src/QsCompiler/CommandLineTool/Options.cs

```diff
@@ -72,6 +72,13 @@ public class CompilationOptions : Options
             HelpText = "Specifies the classical capabilites of the runtime. Determines what QIR profile to compile to.")]
         public AssemblyConstants.RuntimeCapabilities RuntimeCapabilites { get; set; }
 
+        internal RuntimeCapability RuntimeCapability => this.RuntimeCapabilites switch
+        {
+            AssemblyConstants.RuntimeCapabilities.QPRGen0 => RuntimeCapability.BasicQuantumFunctionality,
+            AssemblyConstants.RuntimeCapabilities.QPRGen1 => RuntimeCapability.BasicMeasurementFeedback,
+            _ => RuntimeCapability.FullComputation
+        };
```

Comment:
I was not able to update the capability type in the command-line parser, since the new type is not an enum (CommandLineParser has special parsing for enums). I think this will take a little more time to parse the custom type with proper error handling.

Reply:
I'd suggest to just go with string and use the TryParse method that you added to the type. But I agree to make this a separate PR.